connector/subgraph/src/helpers/index.ts

```diff
@@ -88,6 +87,12 @@ export function getProposalEntity(
       '0x0000000000000000000000000000000000000000'
     )
     proposal.challengeEndDate = BigInt.fromI32(0)
+    proposal.snapshotBlock = BigInt.fromI32(0) // needed because of required value on Voting data
```

Comment:
going to check again but i think that i was having some weird issue if i removed the required from that

Reply:
ah is because is pretty annoying to deal with a ton of ifs when we need the values and check if is not null, this is the error: 
```
ERROR AS200: Conversion from type '~lib/@graphprotocol/graph-ts/index/BigInt | null' to '~lib/@graphprotocol/graph-ts/index/BigInt' requires an explicit cast.

     proposal.snapshotBlock
     ~~~~~~~~~~~~~~~~~~~~~~
 in src/DisputableVoting.ts(124,4)
```

and the error happens here 
https://github.com/1Hive/honey-pot/blob/9660eb0c049b82446a8fd7b6c3870bef837a2f6d/connector/subgraph/src/DisputableVoting.ts#L124